lightstep-tracer-jre/src/test/java/com/lightstep/tracer/jre/JRETracerTest.java

```diff
@@ -8,7 +8,7 @@
 import io.opentracing.Span;
 import io.opentracing.SpanContext;
 import io.opentracing.Tracer;
-import io.opentracing.propagation.TextMapExtractAdapter;
+import io.opentracing.propagation.TextMapAdapter;
```

Comment:
This change seems unrelated to the PR

Reply:
It fixes a test which would break "mvn clean install"

Reply:
I'm not sure I follow. I checked out master, updated OT to 0.32.0 and successfully ran mvn clean install. Maybe if you separate out the OT dependency upgrades from the formatting changes and the common version bumps and the version bumps of this jar it will be more clear?

Reply:
Okay I'll just undo it

Reply:
If updating OT to .32 works